src/format/tests.rs

```diff
@@ -25,6 +25,8 @@ fn module_test() {
     assert_format!("import one/two/three as free\n");
     assert_format!("import one/two/three.{thunk} as free\n");
     assert_format!("import one/two/three.{thunk as funky} as free\n");
+    assert_format!("import my/cool/module.{\nOne, Two, Three, Four, Five, Six, Seven, Eight, Nine, Ten, Eleven, Twelve, \nThirteen, Fifteen, Sixteen\n}\n");
```

Comment:
Could we have some indentation after the break? Currently it is formatted like this:

```rust
import my/cool/module.{
One, Two, Three, Four, Five, Six, Seven, Eight, Nine, Ten, Eleven, Twelve, 
Thirteen, Fifteen, Sixteen
}
```

When I think it should be formatted like this:

```rust
import my/cool/module.{
  One, Two, Three, Four, Five, Six, Seven, Eight, Nine, Ten, Eleven, Twelve, 
  Thirteen, Fifteen, Sixteen,
}
```

There is also some trailing whitespace on the second line.

Reply:
The fix is checked in.